lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -49,26 +49,61 @@ def save_as(path)
         # @return [Xcodeproj::Config]
         #
         def generate
-          pod_targets = target.pod_targets_for_build_configuration(@configuration_name)
           config = {
             'OTHER_LDFLAGS' => '$(inherited) ' + XCConfigHelper.default_ld_flags(target),
             'OTHER_LIBTOOLFLAGS' => '$(OTHER_LDFLAGS)',
             'PODS_ROOT' => target.relative_pods_root,
             'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) COCOAPODS=1',
           }
+          @xcconfig = Xcodeproj::Config.new(config)
+
+          generate_settings_to_import_pod_targets
+
+          XCConfigHelper.add_target_specific_settings(target, @xcconfig)
+
+          generate_vendored_build_settings
+          generate_other_ld_flags
+
+          # TODO: Need to decide how we are going to ensure settings like these
+          # are always excluded from the user's project.
+          #
+          # See https://github.com/CocoaPods/CocoaPods/issues/1216
+          @xcconfig.attributes.delete('USE_HEADERMAP')
```

Comment:
This wasn't introduced. This is just kept in the code as it was after some refactorings.